csharp/ql/lib/semmle/code/csharp/security/cryptography/EncryptionKeyDataFlowQuery.qll

```diff
@@ -66,15 +66,18 @@ class SymmetricEncryptionCreateDecryptorSink extends SymmetricEncryptionKeySink
 /**
  * Symmetric Key Data Flow configuration.
  */
-class SymmetricKeyTaintTrackingConfiguration extends TaintTracking::Configuration {
```

Comment:
You'll need to keep the old config around and deprecate it.

Reply:
Oh right!